### DISCUSSION TITLE
Error while saving a new page

### DISCUSSION THREAD

**Discussion by mx2fr** [requarks/wiki#3365] [Help / Questions]
Hi,

everytime I want to create a new page or save a page an error occurs.
The problem appeard after the update to version 2.2.51
![Screenshot_1](https://user-images.githubusercontent.com/48054902/78261920-2803aa00-7500-11ea-95af-f983949b3ffc.png)

**Comment by NGPixel** [comment]
Do you have a reverse proxy in front of your wiki? This error doesn't seem to be coming from wiki.js

**Comment by mx2fr** [comment]
Yes we got an nginx Webserver in front ob the wiki, the error occurred directly after we updated the wiki to 2.2.51
Am 2. Apr. 2020, 17:23 +0200 schrieb Nicolas Giard <notifications@github.com>:

**Comment by Smankusors** [comment]
can you share your nginx configuration for your wiki?

**Comment by mx2fr** [comment]
Sure, it’s attached.

Greeting Max
Am 2. Apr. 2020, 18:42 +0200 schrieb Antony Kurniawan <notifications@github.com>:

**Comment by Smankusors** [comment]
there's.... no... attachment here :/

**Comment by mx2fr** [comment]
Sorry, thats my stupid mail programm
Now it should be attached.
 

Am 3. April 2020 um 10:32 schrieb Antony Kurniawan <notifications@github.com>:

**Comment by NGPixel** [comment]
Please reply from the GitHub website and not by email. There's still no attachment and it's wasting everybody's time.

**Comment by mx2fr** [comment]
I´m sorry, here is the config.
[wiki.js.txt](https://github.com/Requarks/wiki/files/4427003/wiki.js.txt)

**Comment by mx2fr** [comment]
Did you get my config file?

**Comment by Smankusors** [comment]
well yes, and welp... I can't reproduce this error....

can you give us your host info? How do you run Wiki.js? Docker?

**Comment by mx2fr** [comment]
On a Debian 10 VM with PostgreSQL and Nginx

Am 6. Apr. 2020, 18:12 +0200 schrieb Antony Kurniawan <notifications@github.com>:

**Comment by Smankusors** [comment]
hmm... is there any logs? from nginx? from Wiki.js? from your browser console?

**Comment by mx2fr** [comment]
No logs in the browser console, NGINX logs are attached, where do I find the Wiki.Js logs?
![Screenshot_2](https://user-images.githubusercontent.com/48054902/78589055-e47ea800-783f-11ea-9444-4e19857110e4.png)

**Comment by Smankusors** [comment]
it depends how do you run it. How do you run Wiki.js?

**Comment by mx2fr** [comment]
As a node.js service
Am 6. Apr. 2020, 23:14 +0200 schrieb Antony Kurniawan <notifications@github.com>:

**Comment by Smankusors** [comment]
> node.js service

You followed [this guide](https://docs.requarks.io/install/linux#run-as-service), yes? Then you can read the logs using `journalctl -u wiki` as mentioned from that guide.

**Comment by mx2fr** [comment]
here are the logs
![Screenshot_1](https://user-images.githubusercontent.com/48054902/78755669-d3d24d00-7979-11ea-8d97-f9857be9ee30.png)

**Comment by Smankusors** [comment]
is the file exists on your sftp?

**Comment by mx2fr** [comment]
No, it doesn´t exist on the sftp server

**Comment by mx2fr** [comment]
should I delete all files on the sftp server and let the wiki do a complete new backup?

**Comment by Smankusors** [comment]
hmm... if you disable SFTP storage, is the error still shows up?

**Comment by mx2fr** [comment]
No, now everything works fine
